### ISSUE TITLE
Ignore torrents with less than X seeders

### ISSUE THREAD

**Issue by theCheek** [cardigann/cardigann#75]
So I was thinking of an enhancement which would be incredibly useful, and wanted to put it out there.

Seeing as Sonarr, for reasons only known to the Devs themselves, have not yet implemented a minimum seeders option (do not pick up torrents with say less than 10 seeders), I was wondering if this could be something integrated into Cardigann? So when a request comes in, it only returns torrents that will have a good download speed? 

Obviously not a priority right now during Cardigann's Genesis, but do you think such a feature would be possible?

**Comment by lxt2** [comment]
Are you sure that Sonarr doesn't prioritize torrents with more seeders? 

It would absolutely be possible, I've been meaning to make the Aggregate searcher a bit smarter and add filters.

**Comment by theCheek** [comment]
AFAIK Sonarr  rejects torrents with 0 seeds, but this is not customisable. It does also prioritise based on seeders, but only after many other criteria first such as file size. 

Great the aggregate would then be the only indexer needed to be entered into sonarr, and the frustrating event of sonarr choosing a slow torrent over a high peer one would go away!

**Comment by zapp1** [comment]
I think it is not done in sonarr because if you wanted to pick up a file from an older show or from a show that has series going back a few years then there may only be one or two torrents available with a handful of seeders.

I usually set up my sonarr indexer page tags for popular download teams eg ettv or rartv as these tend to have multiple seeders

**Comment by theCheek** [comment]
@zapp1 Yeah I can understand the logic, but seems a little heavy handed for covering all user setups. For example my Sonarr is setup to search torrents first, but then try Usenet, therefore Sonarr grabs a low-seed torrent in favour of an nzb. I think this modification would get around that issue - but again, should be a toggle.
